### DISCUSSION TITLE
Rename to hanna?

### DISCUSSION THREAD

**Discussion by jeremyevans** [rdoc/hanna-nouveau#22] [General]
@mislav would you be open to assigning the `hanna` gem to me in rubygems.org?  I've been maintaining `hanna-nouveau`, the main fork of `hanna` for many years.  I think it may be simpler to rename `hanna-nouveau` to `hanna`, if for no other reason than it is much easier to spell.

**Comment by mislav** [comment]
For sure! Send me your Rubygems email please.

**Reply by jeremyevans** [reply]
code@jeremyevans.net is my email.  Thanks!

**Reply by jeremyevans** [reply]
Thank you very much!  I'll try to put out a hanna release next week, and an empty hanna-nouveau release that depends on hanna.

**Reply by jeremyevans** [reply]
Sorry for the delay.  I finally had time to put out a release of hanna today.  @mislav, do you want to continue to be an owner of the hanna gem?

**Reply by mislav** [reply]
No thanks! You can remove me.

**Reply by jeremyevans** [reply]
Done.  Thank you very much for starting the hanna project!

**Reply by mislav** [reply]
Thanks for maintaining it! And for switching it away from Haml 😅 💀
